tests/helpers/common/bridge.go

```diff
@@ -51,9 +51,12 @@ func BuildBridge(ctx context.Context, runenv *runtime.RunEnv, initCtx *run.InitC
 	cfg.Gateway.Enabled = true
 	cfg.Gateway.Port = "26659"
 
-	optlOpts := []otlpmetrichttp.Option{
-		otlpmetrichttp.WithEndpoint(runenv.StringParam("otel-collector-address")),
-		otlpmetrichttp.WithInsecure(),
+	var optlOpts []otlpmetrichttp.Option
+	if runenv.StringParam("otel-collector-address") != "" {
```

Comment:
this will be transfered as a separate issue to solve
Idea: we will include metrics as a standard node init (good first issue)

Reply:
will be fixed by #178